forge/src/main/java/whocraft/tardis_refined/registry/forge/ItemRegistryImpl.java

```diff
@@ -1,24 +1,41 @@
 package whocraft.tardis_refined.registry.forge;
 
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.flag.FeatureFlagSet;
 import net.minecraft.world.item.CreativeModeTab;
+import net.minecraft.world.item.CreativeModeTabs;
+import net.minecraft.world.item.Item;
 import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.Blocks;
+import net.minecraftforge.event.CreativeModeTabEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
 import whocraft.tardis_refined.TardisRefined;
 import whocraft.tardis_refined.registry.BlockRegistry;
 import whocraft.tardis_refined.registry.ItemRegistry;
+import whocraft.tardis_refined.registry.RegistrySupplier;
 
+@Mod.EventBusSubscriber(modid = TardisRefined.MODID, bus = Mod.EventBusSubscriber.Bus.MOD)
 public class ItemRegistryImpl {
-
-    public static CreativeModeTab TAB = new CreativeModeTab(TardisRefined.MODID) {
-        @Override
-        public ItemStack makeIcon() {
-            return new ItemStack(BlockRegistry.GLOBAL_CONSOLE_BLOCK.get());
-        }
-    };
+    public static CreativeModeTab TAB;
```

Comment:
This doesn't appear to be initialised but is being referenced by getCreativeTab?

Reply:
It is not initialized until the event is fired, yeah.

Reply:
I appear to have made a mistake, I forgot to give TAB a value when the tab is registered